### DISCUSSION TITLE
How to get control of the performance of server-side rendering

### DISCUSSION THREAD

**Discussion by joeprivettsmart** [remix-run/remix#8990] [Q&A]
We are running remix on our servers. The "waiting for server" time in Chrome Dev tools seems to be very long 2-4 seconds. We would like to be able to understand how we can performance tweak remix. Is this possible to tweak performance, or is it simply closed off to that? How does the Remix team effectively monitor and improve performance of Remix?

**Comment by kiliman** [comment]
I think you'll need to provide more information. That's generally not the experience for must users.

**Comment by sergiodxa** [comment]
You could use [Metronome](https://metronome.sh) or Sentry to track performance.

But usually waiting for server means your loader is slow.
